amazon_cloud_code_generator/cmd/refresh_examples.py

```diff
@@ -145,10 +107,10 @@ def inject(target_dir, extracted_examples):
         new_content = ""
         in_examples_block = False
         for l in module_path.read_text().split("\n"):
-            if l == "EXAMPLES = r'''":
+            if l == 'EXAMPLES = r"""':
```

Comment:
This doesn't match what's in `templates/header.j2`, so nothing is being added to the examples block. Either this change should be reverted or the template needs to be changed.

Reply:
@gravesm Ah, done, thanks! I had some local changes and didn't realise.